.env.regtest

```
@@ -0,0 +1,21 @@
+# monorepo arkade-regtest overrides
+
+# Keep Fulmine pinned away from v0.3.18; boltz-swap observed ARK balance issues there
+FULMINE_IMAGE=ghcr.io/arklabshq/fulmine:v0.3.15
+
+ARKD_IMAGE=ghcr.io/arkade-os/arkd:v0.9.9-rc.0
+ARKD_WALLET_IMAGE=ghcr.io/arkade-os/arkd-wallet:v0.9.9-rc.0
+
+# Shared regtest tuning used across the SDK repos
+ARKD_VTXO_TREE_EXPIRY=20
+ARKD_BOARDING_EXIT_DELAY=40
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The ARKD_BOARDING_EXIT_DELAY key should go before the ARKD_VTXO_TREE_EXPIRY key

(UnorderedKey)